### ISSUE TITLE
Add to Database: Leviton [ID:30b,Type:202]

### ISSUE THREAD

**Issue by stevemac00** [cdjackson/HABmin2#122]
Manufacturer: Leviton
ID: 30b (as reported by binding)
Type: 202  (as reported by binding)
Alliance Link: [http://products.z-wavealliance.org/products/379](url)
Manufacturer Website: [http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=VRP03-1LW&section=44145&minisite=10251](url)

Similar to Leviton VRPD3-1LW which is apparently a replacement product (and is in database).

**Comment by stevemac00** [comment]
I copied leviton/vrpd3-1lw.xml to leviton/vrp03-1lw.xml and added similar entry to products.xml.
Unfortunately, it does not work but I'm not knowledgable enough to see what's wrong.
It keeps looping (see attached log fragment).
[zwave.log.txt](https://github.com/cdjackson/HABmin2/files/119196/zwave.log.txt)

**Comment by cdjackson** [comment]
Isn’t the problem that the device isn’t responding to messages? I can’t see response, but I’ve only had a quick look on my phone…

**Comment by stevemac00** [comment]
Is this not a response? Sorry, I've not learned enough about the protocol to decipher the logs.
`2016-02-05 07:29:10.556 [DEBUG] [b.z.i.protocol.ZWaveController:453 ]- NODE 2: Init node thread start
2016-02-05 07:29:10.560 [DEBUG] [o.b.z.i.protocol.SerialMessage:233 ]- Assembled message buffer = 01 03 00 05 F9 
2016-02-05 07:29:10.564 [DEBUG] [WaveController$ZWaveSendThread:1228]- NODE 255: Sending REQUEST Message = 01 03 00 05 F9 
2016-02-05 07:29:10.569 [DEBUG] [eController$ZWaveReceiveThread:1441]- Receive Message = 01 04 01 05 08 F7 `

**Comment by cdjackson** [comment]
I'm not sure from this bit of log but most of the responses I saw (all of them as far as I could tell from the quick look I had) are responses from the stick, not the device. All the initial requests during the init procedure go to the stick - these are ok, but when we try to communicate with the actual device, all I could see from the quick look I had was timeouts.

**Comment by stevemac00** [comment]
I downloaded indigo and plugged the z-stick into the mac. Indigo recognized the switch on node 2 and I was able to communicate with the switch. I noticed Indigo reported these commands (which is a little 
different than the vrpd3 mentioned above).
`

    <commandClasses>
        <commandClass id="0020" inNIF="false"/>
        <commandClass id="0026"/>
        <commandClass id="0027"/>
        <commandClass id="002b"/>
        <commandClass id="002c"/>
        <commandClass id="0072"/>
      </commandClasses>
`

**Comment by cdjackson** [comment]
It looks like it does respond ok to the first message, but it's not responding to one of the messages. Can you please provide links to the device information - the ones above don't work...

**Comment by stevemac00** [comment]
[http://products.z-wavealliance.org/products/379](http://products.z-wavealliance.org/products/379)

[http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=VRP03-1LW&section=44145&minisite=10251](http://www.leviton.com/OA_HTML/ProductDetail.jsp?partnumber=VRP03-1LW&section=44145&minisite=10251)

**Comment by cdjackson** [comment]
Thanks.

I’m not sure what’s up. The switch isn’t responding to the version request for the multi_level switch class. Given that there are multiple versions of this class (3 that I know of) it’s important to know what version is supported. In theory we could add something to the database to bypass this, but since the database isn’t read at this part of the initialisation sequence this isn’t straight forward and I don’t have the time to spend on this at the moment as I’m trying to get the OH2 binding released.

**Comment by cdjackson** [comment]
#182 once fixed can be used here - this seems to be a Leviton issue.

**Comment by mhilbush** [comment]
I agree with Chris.  I'd suggest you rediscover this device using a current build (2016-06-19 or later) and see what happens.  If you see a bunch of COMMAND_CLASS_VERSION_GET errors, we may need to add the FORCEVERSION to several of the command classes to stop those.  I could do that now, but I'd rather do it only if necessary.
